test-collateral/generate-policy/src/main.rs

```diff
@@ -16,6 +16,7 @@ use std::{
     path::{Path, PathBuf},
     process::{exit, Command},
     str::FromStr,
+    convert::TryFrom,
```

Comment:
Can you keep the imports in alphabetical order?

Reply:
I need to run rust fmt, I'll manually reorder these if that doesn't